lua/entities/gmod_wire_expression2/core/entity.lua

```diff
@@ -511,6 +511,44 @@ e2function void entity:applyAngForce(angle angForce)
 	end
 end
 
+e2function void entity:applyOffsetAngForce(angle angForce, vector position)
```

Comment:
Unless that's a local position and not a world-grid offset, that 2nd parameter should be named "offset", not "position".